controllers/limitador_controller.go

```diff
@@ -266,6 +291,17 @@ func (r *LimitadorReconciler) reconcileLimitsConfigMap(ctx context.Context, limi
 		return err
 	}
 
+	oldConfigMap := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      strings.TrimPrefix(limitador.LimitsConfigMapName(limitadorObj), "limitador-"),
+			Namespace: limitadorObj.Namespace,
+		},
+	}
+
+	if err := r.Client().Delete(ctx, oldConfigMap); err != nil && !errors.IsNotFound(err) {
+		return err
+	}
```

Comment:
As this code is only required for upgrading to this version it would be good to wrap the code in a todo label to be stating when this code can be removed.

Reply:
As suggested I added TODO: comments and on top of that enclosed the whole block of code inside `{     }` to make it clear what needs to be removed when we no longer need the upgrade path. Not sure if it is a good practice.